Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -992,8 +994,9 @@ webview.getSettings().setJavaScriptEnabled(true);
 
 This allows the WebView to interpret JavaScript. It should be enabled only if necessary to reduce the attack surface to the app. If JavaScript is necessary, you should make sure that
 
-- the communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission
+- The communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission
 - JavaScript and HTML are loaded locally, from within the app data directory or from trusted web servers only.
+- The user cannot define which sources to load by means of loading different resources based on a user provided input.
```

Comment:
I tried a different re-phrasing for this based on your additions below. 

```suggestion
- The WebView's URI cannot be manipulated by the user in order to load different resources than the ones provided by the app.
```

Reply:
It can be more than that. Let's just resolve

Reply:
It can be more than that. Let's just resolve